admin/views/access-arrangements/overview.ejs

```diff
@@ -89,7 +89,7 @@
                             class="panel panel-border-narrow green-panel"  style="<%= VerticalBarStyle %>"
                         <% } %>>
                         <div id="<%= pupil.urlSlug %>">
-                            <% if (hdfSubmitted) { %>
+                            <% if (hdfSubmitted || pupil.hasCompletedCheck || pupil.notTakingCheck) { %>
```

Comment:
This view seems to be getting a lot of business knowledge.  From the perspective of the view the logic is "should I show the link to the pupils' access arrangements or not", and yet here it is making what seems to be a business decision in the view.